### PR TITLE
Allow nullable customTemplate in Configuration setter

### PR DESCRIPTION
**Pull request by PapsOu**
Related to this discussion https://github.com/doctrine/DoctrineMigrationsBundle/commit/11459465d5049812ea83003a761b9787bf01ec0f#commitcomment-24042795

### CODE REVIEW

File: lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php [hunk 1]
```diff
@@ -387,7 +387,7 @@ public function getCustomTemplate() : ?string
      *
      * @param string $customTemplate The custom template path
      */
-    public function setCustomTemplate(string $customTemplate) : void
+    public function setCustomTemplate(string $customTemplate = null) : void
```
Comment (Majkl578) [inline lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php]: Should be `?string`.

**Comment by PapsOu** [comment]
@Majkl578 thanks I forgot that nullable type was applicable also on method parameters.

**Comment by lcobucci** [comment]
@PapsOu it's good now, can you please cover that scenario with a test to ensure we don't break it in the future?

**Comment by PapsOu** [comment]
Ok I will do it this week

**Comment by celowsky** [comment]
The issue that Scrutinizer is reporting does not seem valid. The code style complaint regard the PSR-2 is made on code that seems to be following that standard.

**Comment by Majkl578** [comment]
Don't worry about Scrutinizer, sometimes it can be annoying. 😉 Simple test is still needed though...

**Comment by PapsOu** [comment]
I don't know if the test case is well writen (I'm not really conformtable with Unit testing).

**Comment by lcobucci** [comment]
@PapsOu I had some time now and I've managed to simplify a bit your test. Thanks a lot for the contribution!

**Pull request merged**